src/main/java/mat/client/shared/MeasureLibraryGridToolbar.java

```diff
@@ -49,6 +55,13 @@ void applyDefault() {
         buildButton(exportButton, IconType.DOWNLOAD, "Export", "Click to export", "72px");
     }
 
+    public void addFhirValidationButton(){
+        if (MatContext.get().getFeatureFlagStatus(FeatureFlagConstant.FHIR_CONV_V1)) {
```

Comment:
why cannot we call it from appluDefault() ?
what is the aim of a separate public method call?

Reply:
Test cases are failing if the getFeatureFlagStatus is called from constructor (applyDefault) because of toolbar being Injected prior to that. 

![image](https://user-images.githubusercontent.com/57495404/71601370-29988d00-2b21-11ea-9a2e-253f62fdafda.png)